src/app/Main/Body/InfoPage/InfoPage.model.ts

```diff
@@ -1,5 +1,19 @@
+export interface AmrsTableData {
+    substanceClass: string;
+    amrSubstance: string;
+    cutOffOne: string;
+    cutOffTwo?: string;
+    concentrationMin: string;
+    concentrationMax: string;
+}
 
-export type AmrsTableData = Record<string, string>
+export type AmrsTableDataKey =

```

Comment:
This type is obsolete, there is a built in type: `keyof AmrsTableData`

Reply:
Done. Commit "Remove key type & define dialog outside map"